### ISSUE TITLE
lwk_wasm demo app returns error along with too many requests

### ISSUE THREAD

**Issue by s373nZ** [Blockstream/lwk#64]
Tried out the `lwk_wasm` bindings in a NextJS application today, but got some WASM errors when calling `client.fullScan()`.

Decided to use the demo app for reference, but when trying it out both locally and hosted at:

https://blockstream.github.io/lwk/

It returns an error for both Mainnet and Testnet when scanning the pre-populated CT descriptor:
```
Wollet(Reqwest(reqwest::Error { kind: Request, source: "JsValue(TypeError: Failed to fetch\nTypeError: Failed to fetch\n at __wbg_fetch_27eb4c0a08a9ca04 (webpack:///../pkg/lwk_wasm_bg.js?:2261:17)\n at https://blockstream.github.io/lwk/f387b2b5a5973a6b9e8c.module.wasm:wasm-function[173]:0x9a2c9\n at https://blockstream.github.io/lwk/f387b2b5a5973a6b9e8c.module.wasm:wasm-function[248]:0xde4cb\n at https://blockstream.github.io/lwk/f387b2b5a5973a6b9e8c.module.wasm:wasm-function[372]:0x13c359\n at https://blockstream.github.io/lwk/f387b2b5a5973a6b9e8c.module.wasm:wasm-function[890]:0x1a19ad\n at https://blockstream.github.io/lwk/f387b2b5a5973a6b9e8c.module.wasm:wasm-function[4563]:0x2c2330\n at __wbg_adapter_34 (webpack:///../pkg/lwk_wasm_bg.js?:343:10)\n at real (webpack:///../pkg/lwk_wasm_bg.js?:328:20))" }))
```

There are also about 21 requests to Esplora for various `txs` information, but finally error in a `429 Too Many Requests`.

Additionally, it looks like there is a conflict with the CORS policy, causing this error in the console:

```
Access to fetch at 'https://blockstream.info/liquidtestnet/api/scripthash/967f1d64891b0241ef4bb030c8d78d04f23ec9e27c235e025d955e224aa428b1/txs' from origin 'https://blockstream.github.io' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

It would be nice to have this operational for reference. LMK if you'd be interested in a separate ticket demonstrating the WASM error from my prototype code running serverside as well.

**Comment by RCasatta** [comment]
Hello, sorry I missed this...

I am afraid we are not keeping the demo app updated with latest changes...

Latest versions handle the 429 by retrying with delay.
The CORS error seems a server problem which I am investigating.

In the meantime, did you see https://github.com/RCasatta/liquid-web-wallet which is running at https://liquidwebwallet.org/ ?

**Comment by s373nZ** [comment]
Thanks. I've seen `liquid-web-wallet` frontend under a demo context but didn't think to check out the code til you pointed it out. At first glance, it doesn't seem to be as straightforward as the `lwk_wasm` package to look for patterns, but maybe a closer inspection can help me figure out if issues utilizing `lwk_wasm` are on my end.

I'll leave this issue open as the `lwk_wasm` demo app should be maintained if its in the codebase and documentation, but understandable if it is a low priority.

**Comment by RCasatta** [comment]
We decided to remove the demo app from the codebase and point the users to the wallet app which is a more complete example showing many more features of LWK

**Comment by s373nZ** [comment]
Understandable move :wink: Thanks for the help!
